Demo/Demo/SwiftUIComponents/PayPalWeb/PayPalWebCreateOrderView.swift

```diff
@@ -1,13 +1,11 @@
 import SwiftUI
 
-struct CreateOrderPayPalWebView: View {
+struct PayPalWebCreateOrderView: View {
```

Comment:
We may be able to reuse some sort of create order view across all of the features - left separate for now but could be something to consider as we continue this refactor

Reply:
Agreed!